test/internal/apiserver/BUILD.bazel

```diff
@@ -6,6 +6,11 @@ go_library(
         "apiserver.go",
         "paths.go",
     ],
+    data = [
+        "//hack/bin:com_coreos_etcd",
+        "//hack/bin:io_kubernetes_kube-apiserver",
+        "//hack/bin:kubectl",
+    ],
```

Comment:
this dependency was incorrectly placed in `test/integration/framework/BUILD.bazel`